lib/src/utils/menu.dart

```diff
@@ -55,9 +55,16 @@ class PullDownMenu extends StatelessWidget {
             borderRadius: theme.borderRadius!,
             child: FadeTransition(
               opacity: animation,
-              child: _MenuBody(
-                width: theme.width!,
-                children: items,
+              child: SizedBox(
+                width: theme.width,
```

Comment:
Any reasoning on `SizedBox` here? `_MenuBody` already has `ConstrainedBox` for width configurations

Reply:
Without it, the SizeTransition does not work correctly.
We can use ConstrainedBox instead of SizedBox and remove it from _MenuBody.

https://user-images.githubusercontent.com/42691184/221403050-a09cafe8-3400-4ab1-bbd5-eed5c690bd9f.mp

Reply:
> We can use ConstrainedBox instead of SizedBox and remove it from _MenuBody.

Sounds good to me